### PR TITLE
Issue 330: Adds glossary entries for 'exclude list' and 'excludelist'

### PR DESCRIPTION
**Pull request by bredamc**
Proposed fix for Issue 330.

### CODE REVIEW

**Review by mportman12 — COMMENTED** [review]
LGTM

File: supplementary_style_guide/glossary_terms_conventions/general_conventions/e.adoc [hunk 1]
```diff
@@ -154,6 +154,29 @@ If referring to the program, use "Emacs", for example, "Source-Navigator support
 
 *See also*:
 
+[discrete]
+[[exclude-list]]
+==== image:images/yes.png[with caution] exclude list (noun)
+*Description*: Use _exclude list_ as a synonym for _blocklist_ and _denylist_.
```
[the remark this answers is not in the document]
Comment (jafiala) [reply supplementary_style_guide/glossary_terms_conventions/general_conventions/e.adoc:160]: To me this seems confusing and contrary to my proposal that I originally raised. Readers probably will expect all _-list_ words to be spelled the same, so _blocklist_, _allowlist_, _denylist_, _excludelist_. The proposed standard would be counterintuitive and introduce inconsistency.

[the remark this answers is not in the document]
Comment (dfitzmau) [reply supplementary_style_guide/glossary_terms_conventions/general_conventions/e.adoc:160]: Maybe some engineering system output will determine the through name of this noun? 

I agree with the previous comments.

**Review by dfitzmau — DISMISSED** [review]

**Comment by mportman12** [comment]
@bredamc What's the status of this PR? Do we know what the current blocker is and do we have a plan on how to resolve it?

**Review by joaedwar — APPROVED** [review]
lgtm

**Comment by bergerhoffer** [comment]
@mportman12 This PR is on hold awaiting more discussion. I don't think there was consensus on whether "excludelist" should be one word or two. I think it was originally proposed as two words, but then considering if it should be just one to be consistent with the other options (denylist, allowlist, etc.)

Probably something we can add to next month's meeting

**Comment by mportman12** [comment]
We did not have time to discuss this issue at the September Style Council meeting.

**Pull request closed without merging**